.env.sample

```diff
@@ -52,9 +52,17 @@
 # (default=5)
 # RELAY_THROTTLE_LIMIT=
 # The API key to be used per chain.
-# RELAY_PROVIDER_API_KEY_ARBITRUM_ONE=
-# RELAY_PROVIDER_API_KEY_GNOSIS_CHAIN=
-# RELAY_PROVIDER_API_KEY_SEPOLIA=
+# RELAY_PROVIDER_API_KEY_OPTIMISM
+# RELAY_PROVIDER_API_KEY_BSC
+# RELAY_PROVIDER_API_KEY_GNOSIS_CHAIN
+# RELAY_PROVIDER_API_KEY_POLYGON
+# RELAY_PROVIDER_API_KEY_POLYGON_ZKEVM
+# RELAY_PROVIDER_API_KEY_BASE
+# RELAY_PROVIDER_API_KEY_ARBITRUM_ONE
+# RELAY_PROVIDER_API_KEY_AVALANCHE
+# RELAY_PROVIDER_API_KEY_LINEA
+# RELAY_PROVIDER_API_KEY_BLAST
+# RELAY_PROVIDER_API_KEY_SEPOLIA
```

Comment:
Nit: I'd include the `=` at the end of each key to make explicit they need a value to be set.

Reply:
Added in 1e2cebedfaf6ebea44ca6e1ebf753ca6ea521c88.

Reply:
Nit: I'd suggest also including `""` around strings to make it explicit which variables are numeric and which are strings.